qemu/cfg/seabios_iommu.cfg

```diff
@@ -0,0 +1,39 @@
+# Please add "iommu=pt intel_iommu=on" to kernel line in host
+- seabios_iommu: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = seabios_iommu
+    only q35
+    no WinXP WinVista Win7 Win8 Win8.1 Win2003
+    no Win2008 Win2008..r2 Win2012 Win2012..r2
+    virtio_dev_disable_legacy = on
+    virtio_dev_disable_modern = off
```

Comment:
@ybduan  do we need to add "disable_legacy, disable_modern"?
I think the default virtio driver now is 1.0

Reply:
Modified, Thanks!